src/adera/Machines/RCSController.cpp

```diff
@@ -0,0 +1,125 @@
+#include "RCSController.h"
+#include "osp/Active/ActiveScene.h"
+
+#include <Magnum/Math/Vector.h>
+
+using namespace adera::active::machines;
+using namespace osp::active;
+using Magnum::Vector3;
+
+
+MachineRCSController::MachineRCSController()
+    : Machine(true),
+    m_wiCommandOrient(this, "Orient"),
+    m_woThrottle(this, "Throttle")
+{
+    using wiretype::Percent;
```

Comment:
That's a lot of using declarations....

Really necessary for one line?

Reply:
What's a lot? Yeah here it's not necessary, but I'm curious what others you're referring to. The ones at the top of the file are pretty necessary.

Reply:
Well. For my work projects I basically never use the using keyword to import things out of other namespaces into the current file.

The only exception is when I need to pull something from another namespace to make argument dependent lookup work (because there is no other way to do it. Thanks c++!!)

So a lot is "more than zero".

But both you and cap seem to like it, so when in Rome I guess.